panel/config.py

```diff
@@ -454,11 +459,11 @@ def __getattribute__(self, attr):
             curdoc and attr not in session_config[curdoc]):
             new_obj = copy.copy(super().__getattribute__(attr))
             setattr(self, attr, new_obj)
-        if attr in global_params or attr == 'theme':
+        if attr in _CONFIG_GLOBAL_PARAMS or attr == 'theme':
             return super().__getattribute__(attr)
         elif curdoc and curdoc in session_config and attr in session_config[curdoc]:
             return session_config[curdoc][attr]
-        elif f'_{attr}' in params and getattr(self, f'_{attr}_') is not None:
+        elif f'_{attr}' in _params and getattr(self, f'_{attr}_') is not None:
             return super().__getattribute__(f'_{attr}_')
         return super().__getattribute__(attr)
 
```

Comment:
If noticed a huge, huge speed up if I change from `self._admin` to `_config.admin` in the property.

![image](https://github.com/holoviz/panel/assets/42288570/ec366d46-a14b-451d-98f1-55769e1c17ec)

I don't understand why and I did not change it.

But you can see the decrease below

![image](https://github.com/holoviz/panel/assets/42288570/30271cbf-36a4-4020-b98d-9724c9ee3d7d)

Reply:
`admin` should have been declared as a global config option.